### ISSUE TITLE
Replication controller - Resize dialog should have warning / limit

### ISSUE THREAD

**Issue by davsclaus** [hawtio/hawtio-kubernetes#36]
eg we should have some kind of warning / confirm if you resize = 0. 
And also the ui should likely be limited to not allow resize higher than 100 or ? eg so you cannot resize = 9999999

**Comment by davsclaus** [comment]
not sure if OS has some limit already in the osc CLI. Could be worth to see and align what they allow

**Comment by jstrachan** [comment]
I think there's per project quotas that we could enforce the resize to be within

**Comment by davsclaus** [comment]
Ah okay, but I guess if we try to resize above the quota then it would fail on server side? Just wonder if there is a gentle way of knowing the quote?

**Comment by jstrachan** [comment]
Yeah - it'd be ideal if the UI could show the quota (and validate that you don't try to resize over your quota)
